frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -669,3 +669,13 @@ export { useModal } from '../app/modal-support/useModal';
  */
 export const ActionServiceProvider: React.FC<ActionServiceProviderProps> = require('@console/shared/src/components/actions/ActionServiceProvider')
   .default;
+
+/**
+ * A component that renders a namespace dropdown menu.
+ * @param {boolean} hideProjects - (optional) A boolean flag that hides project names in the dropdown if set to true. Defaults to false.
```

Comment:
I think we dont have to expose this right now. Looking at the implementation, it seems like a weird prop to have as it basically wont render the `NamespaceBar` when set to true..

Reply:
Agreed. I removed the documentation and didn't include it in the exposed props type per your suggestion.